sound/soc/sof/sof-priv.h

```diff
@@ -151,11 +142,25 @@ struct snd_sof_dsp_ops {
 	snd_pcm_uframes_t (*pcm_pointer)(struct snd_sof_dev *sdev,
 					 struct snd_pcm_substream *substream);
 
-	/* FW loading */
-	int (*load_firmware)(struct snd_sof_dev *sof_dev);
-	int (*load_module)(struct snd_sof_dev *sof_dev,
-			   struct snd_sof_mod_hdr *hdr);
-	int (*fw_ready)(struct snd_sof_dev *sdev, u32 msg_id);
```

Comment:
IMO, stall, core_power_up/down should be optional, and write, read, write64, read64, block_read, block_write are same for all our platforms so let's remove them from struct snd_sof_dsp_ops.
Others looks fine.

Reply:
it's not because we use the same read/write/write64/read64 that we should remove the abstraction in ops. @dbaluta's last PR shows there could be different ones for other platforms.